### PR TITLE
add assertions for existing files and not matching regexes

### PR DESCRIPTION
**Pull request by mimmi20**
I would like to add two new assertions I want to use.

### CODE REVIEW

File: lib/Assert/Assertion.php [hunk 1]
```diff
@@ -737,6 +743,34 @@ public static function regex($value, $pattern, $message = null, $propertyPath =
...
+     *
+     * @throws \Assert\AssertionFailedException
+     */
+    public static function notRegex($value, $pattern, $message = null, $propertyPath = null)
+    {
+        static::string($value, $message, $propertyPath);
+
+        if (\preg_match($pattern, $value)) {
+            $message = \sprintf(
+                static::generateMessage($message ?: 'Value "%s" does match expression.'),
```
Comment (rquadling) [inline lib/Assert/Assertion.php]: 'does match' => 'matches'

File: lib/Assert/Assertion.php [hunk 2]
```diff
@@ -1468,6 +1502,33 @@ public static function directory($value, $message = null, $propertyPath = null)
...
+     *
+     * @throws \Assert\AssertionFailedException
+     */
+    public static function fileExists($value, $message = null, $propertyPath = null)
+    {
+        static::string($value, $message, $propertyPath);
+
+        if (!\file_exists($value)) {
+            $message = \sprintf(
+                static::generateMessage($message ?: 'Path "%s" was expected to be exsting.'),
```
Comment (rquadling) [inline lib/Assert/Assertion.php]: Assert::file() covers this assertion.

File: README.md [hunk 3]
```diff
@@ -204,6 +204,7 @@ Assertion::extensionLoaded(mixed $value);
 Assertion::extensionVersion(string $extension, string $operator, mixed $version);
 Assertion::false(mixed $value);
 Assertion::file(string $value);
+Assertion::fileExists(string $value);
```
[the remark this answers is not in the document]
Comment (mimmi20) [reply README.md]: Done

**Comment by rquadling** [comment]
Thank you.

**Pull request merged**